brave/browser-laptop
# Preferences info buttons should now have tooltips - fixes issue #8305

Resolves #8305

## Submitter Checklist:


## Test Plan:
1. Start Brave -> Preferences -> Plugins
2. Ensure top 2 Flash tooltips are shown
3. Go to Preferences -> Sync
4. Ensure Sync tooltip is shown

## Reviewer Checklist:


Tests




Preferences info buttons should now have tooltips fixes issue #8305
